src/quantum/setup.py

```diff
@@ -17,7 +17,7 @@
 # This version should match the latest entry in HISTORY.rst
 # Also, when updating this, please review the version used by the extension to
 # submit requests, which can be found at './azext_quantum/__init__.py'
-VERSION = '0.15.0'
```

Comment:
This change will trigger a release.

Reply:
Please don't release from this PR.  I will restore the 0.15.0 version.